soccer/gameplay/plays/sequences/move_sequence.py

```diff
@@ -0,0 +1,25 @@
+import single_robot_sequence
+import role_assignment
+import skills.move
+import behavior
+
+#Moves a SINGLE robot in a sequence of points
+class MoveSequence(single_robot_sequence.SingleRobotSequence):
```

Comment:
Gotcha, makes sense. Just wanted to point out that the behavior sequence should be able to be used without creating a subclass for each type of behavior contained (i.e. Move). It would be great if BehaviorSequence could be used directly for things like `BehaviorSequence([LineUp(), PivotKick(), VictoryDance()])` without having to create a subclass.

From a brief skim of our plays/behaviors, I see a few places that could benefit from using sequences. One that you could probably do is RepeatedLineUp. It could look something like this:

~~~
sequence = BehaviorSequence([
  tactics.LineUp(generate_line(left)),
  Delay(1),
  tactics.lineUp(generate_line(right)),
  Delay(1)])
sequence.repeat = True
add_subbehavior(sequence)
~~~

The Delay behavior doesn't actually exist yet, but it would be easy to create a behavior that takes a timeout value and enters the completed state once the time elapses.

Another play that would actually be a great candidate for sequences is StressTest. It's basically equivalent to your MoveTest. Also take a look at TestCapture.

One (semi-related) thought I have from browsing through plays is that you could create another behavior similar to sequence, but that runs behaviors in parallel (maybe call it "BehaviorGroup"?). This would be a fairly simple wrapper class that just adds all of the given behaviors as subbehaviors then changes its state based on the subbehaviors' states. When all subbehaviors complete, it transitions to completed. If a subbehavior "un completes", it transitions to the running state. This could replace a lot of the duplicated logic we have throughout our gameplay code for doing this. A few behaviors that do this include: tactics.LineUp, stopped.CircleNearBall, and stopped.CircleOnCenter. Just a thought - it may be more trouble than it's worth.

I could see it being more useful for silly demos such as:
~~~
BehaviorSequence([
  BehaviorGroup([Move(pos) for pos in POINTS_FORMING_LETTER_G]),
  BehaviorGroup([Move(pos) for pos in POINTS_FORMING_LETTER_T])])
~~~